modules/nw-networkpolicy-create.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// networking/configuring-networkpolicy.adoc
+
+[id="nw-networkpolicy-create-{context}"]
+
+= Creating a `NetworkPolicy` object
+
+To define granular rules describing ingress network traffic allowed for projects
+in your cluster, you can create `NetworkPolicy` objects.
+
+.Prerequisites
+
+* A cluster using the OpenShiftSDN network plug-in with `mode: NetworkPolicy`
+set.
+* The `oc` CLI program must be installed.
+* You must be logged in to the cluster and your user must have the `admin` role
+for the project you are creating policy objects in.
+
+.Procedure
+
+. Create a policy rule:
+** Create a `<policy-name>.yaml` file where `<policy-name>` describes the policy
+rule.
+** In the file you just created define a policy object, such as in the following
+example:
++
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: <policy-name> <1>
+spec:
+  podSelector:
+  ingress: []
+----
+<1> Specify a name for the policy object.
+
+. Run the following command to create the policy object:
++
+----
+$ oc create -f <policy-name>.yaml -n <project>
+----
++
+In the following example, a new `NetworkPolicy` object is created in a project
+named `project1`:
+----
```

Comment:
Need a "+" in the line above this, so that the following code is indented properly with step 2.